src/main/java/dao/AccountDao.java

```diff
@@ -0,0 +1,132 @@
+package dao;
+
+import card.Card;
+import org.sqlite.SQLiteDataSource;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Optional;
+
+public class AccountDao implements Dao<Account> {
+
+    private static final String CREATE_TABLE_QUERY = """
+            CREATE TABLE IF NOT EXISTS card(
```

Comment:
In the domain, you are using the `Account` model, but the `card` table contains the same data. Can you normalize it?

Reply:
Should I rename the `card` table to the `account`?

Reply:
Renamed the `card` table to the `account`